concepticondata/conceptlists/KBA-2021-912.tsv

```diff
@@ -0,0 +1,913 @@
+KBA-2021-912-489	489	m348-k-bp-fl-fa63	Scorpion (sp.)	1538	SCORPION		crabe, langoustine	Krabbe; Skorpion	krap; skerpioen	kankarapa (crab)	(Scorpiones sp.)	4CC. Undomesticated Animals	M327		
+KBA-2021-912-490	490	m349-k-bp-fl-fa	crocodile	1857	CROCODILE		crocodile	Krokodil	krokodil	kwena		4CC. Undomesticated Animals	M328		
+KBA-2021-912-491	491	m35-k-bp-fl-fa	voice	1363	VOICE		voix	Stimme	stem	kodu, di-; lentswe, ma-		1B. Communication	M134		
+KBA-2021-912-492	492	m350-k-bp-fl-fa	python	1563	PYTHON		python	Python	piton, luislang	lebolobolo		4CC. Undomesticated Animals	M329		
+KBA-2021-912-493	493	m351-k-bp-fl-fa	snake	730	SNAKE		serpent, couleuvre	Schlange	slang	noga		4CC. Undomesticated Animals	M330	s144	
+KBA-2021-912-494	494	m352-k-bp-fl-fa	lizard	632	LIZARD		lézard	Eidechse	akkedis	mokgantitswane		4CC. Undomesticated Animals	M331		
+KBA-2021-912-495	495	m353-k-bp-fl-fa	snail	1543	SNAIL		escargot	Schnecke	slak	kgopa		4CC. Undomesticated Animals	M332		
+KBA-2021-912-496	496	m354-k-bp-fl-fa	frog	503	FROG		grenouille	Frosch	padda	segogwane		4CC. Undomesticated Animals	M333		
+KBA-2021-912-497	497	m355-k-bp-fl-fa	worm	1219	WORM		ver	Wurm	wurm	seboko		4CC. Undomesticated Animals	M334	s197	
+KBA-2021-912-498	498	m356-k-bp-fl-fa67	Bee (Honey bee)	665	BEE		abeille	Biene, Honigbiene	by	notsi	(Apis mellifera)	4CC. Undomesticated Animals	M335		
+KBA-2021-912-499	499	m357-k-bp-fl-fa	ant, soldier ant	587	ANT		fourmi	Ameise	mier	tshoswane		4CC. Undomesticated Animals	M336		lj071
+KBA-2021-912-500	500	m358-k-bp-fl-fa	termite, white ant	883	TERMITE		termite, fourmi blanche	Termite	rysmier	motlhwa		4CC. Undomesticated Animals	M337		
+KBA-2021-912-501	501	m359-k-bp-fl-fa	fly, house-fly (insect)	1504	FLY (INSECT)		mouche	Fliege, Hausfliege	vlieg	ntsi		4CC. Undomesticated Animals	M338	s054	lj020
+KBA-2021-912-502	502	m359-k-bp-fl-fa69	Fly (Housefly)	1504	FLY (INSECT)		mouche	Fliege, Hausfliege	vlieg	ntsi	(Musca domestica)	4CC. Undomesticated Animals	M338	s054	lj020
+KBA-2021-912-503	503	m36-k-bp-fl-fa	to shout (call, cry out)	715	SHOUT		crier, écrier	schreien, rufen	skreeu	-gala; -goa		1B. Communication	M135		
+KBA-2021-912-504	504	m360-k-bp-fl-fa	mosquito	1509	MOSQUITO		moustique	Stechmücke, Moskito	muskiet	monang		4CC. Undomesticated Animals	M339		
+KBA-2021-912-505	505	m361-k-bp-fl-fa	spider	843	SPIDER		araignée	Spinne	spinnekop	segokgo		4CC. Undomesticated Animals	M340		
+KBA-2021-912-506	506	m362-k-bp-fl-fa70	Louse (Head louse)	310	HEAD LOUSE		pou	Laus (Kopflaus, Körperlaus)	luis	nta	(Pediculus humanus capitis)	4CC. Undomesticated Animals	M341	s093	lj015
+KBA-2021-912-507	507	m363-k-bp-fl-fa	bird	937	BIRD		oiseau	Vogel	voël	nonyane		4CC. Undomesticated Animals	M342	s012	lj091
+KBA-2021-912-508	508	m364-k-bp-fl-fa43	Helmeted guinea-fowl				pintade	Perlhuhn	tarentaal	kgaka	(Numida meleagris)	4CC. Undomesticated Animals	M346		
+KBA-2021-912-509	509	m365-k-bp-fl-fa	hawk (harrier)	3258	FALCON		faucon	Falke	sperwel o. valk	ntsu		4CC. Undomesticated Animals	M347		
```

Comment:
This is a bit confusing as the mapping is based on the translations in the other languages. It also appears to be a distinct hawk species (see African harrier-hawk).